packages/datetime/src/timezone/timezoneItem.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import { IconName } from "@blueprintjs/core";
+
+/** Timezone-specific QueryList item */
+export interface ITimezoneItem {
+    /**
+     * The actual timezone name, like `Europe/Paris`.
+     *
+     * This property is used as the identifier for a timezone item, so it should be unique.
+     * If you can't guarantee uniqueness, the `key` property can override it.
+     */
+    timezone: string;
+
+    /** Display name for this timezone. Defaults to `timezone` property. */
+    displayName?: string;
+
+    /** Optional icon for the timezone. */
```

Comment:
what's an example use case for this?

Reply:
`locate` for current timezone

Reply:
can we constrain the API here? I don't think it's necessary to allow any arbitrary icon. just ask the user to supply the current timezone (or detect it by default using native APIs) and show the `locate` icon yourself

Reply:
yeah considered keeping around the `showCurrentTimezone` option. will see if i can bring it back. i removed cuz it was awkward to force an item in there that might look totally different from user-supplied items (like, it's in english but they gave me french items).